common/web-socket-client/web-socket-client.ts

```diff
@@ -32,6 +32,14 @@ export interface LoadSubtitlesCommand {
     };
 }
 
+export interface SyncTimestampCommand {
+    command: 'sync-timestamp';
```

Comment:
Can you change this to 'seek' instead? It is a little clearer IMO.

Reply:
Just pushed a commit to refactor sync into seek